O/object_store_ffi/build_tarballs.jl

```diff
@@ -36,5 +36,5 @@ dependencies = Dependency[
 # Build the tarballs
 build_tarballs(
     ARGS, name, version, sources, script, platforms, products, dependencies;
-    compilers=[:c, :rust], julia_compat="1.6", preferred_gcc_version=v"5",
+    compilers=[:c, :rust], julia_compat="1.9", preferred_gcc_version=v"5",
```

Comment:
Ah, you require Julia v1.9.  We run CI here with Julia v1.7 and can't upgrade that version, so I think your only option is to set `dont_dlopen=true` for the [`LibraryProduct`](https://docs.binarybuilder.org/stable/reference/#BinaryBuilderBase.LibraryProduct) above.

Reply:
Done!

Reply:
Aaah, we have a less "aggressive" `dont_dlopen` option to pass the `build_tarballs` instead of `LibraryProduct`

Reply:
```suggestion
    compilers=[:c, :rust], julia_compat="1.9", preferred_gcc_version=v"5", dont_dlopen=true,
```